packages/server/lib/html/spec-bridge-iframe.html

```diff
@@ -6,7 +6,9 @@
   </head>
   <body>
     <script type="text/javascript">
-      document.domain = '{{domain}}';
+  {{if(options.documentSuperDomainIfExists)}}
```

Comment:
should this be four spaces?

Reply:
The template looks ugly but that seems to be the way to get it displayed correctly? 
![](https://user-images.githubusercontent.com/3980464/210620414-41fb8aa5-89e2-4af2-a3e8-b0b2e728ce05.png)

The [docs](https://v7--squirrellyjs.netlify.app/docs/v7/cheatsheet#conditionals) didn't really explain it but the template also seems to add spacing?